doc/changes/unreleased.md

```diff
@@ -13,6 +13,10 @@
 
 * #33: Updated to Python 3.10
 
+## Documentation
+
+* Update project documentation
+
 ## Internal
```

Comment:
"Internal" is not among the established categories AFAIK.
For updating dependencies you could either choose "security" or "refactoring". 
After each headline there should be an empty line. 
In all other repos we prefer to use past tense. 
Please try to add a reference to a ticket, if there is such.
If not, then maybe we could think of creating such tickets in future before implementing changes?